baseimages/docker-cli/docker.Windows.Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Required.
+# docker build -f baseimages/docker.Windows.Dockerfile -t docker .
+ARG WINDOWS_IMAGE=microsoft/windowsservercore:1803
+
+FROM ${WINDOWS_IMAGE} as download
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV DOCKER_VERSION 18.03.0-ce
+
+# Utility to extract the Docker toolbox zip
+RUN Invoke-WebRequest 'http://constexpr.org/innoextract/files/innoextract-1.6-windows.zip' -OutFile 'innoextract.zip' -UseBasicParsing ; \
+    Expand-Archive innoextract.zip -DestinationPath C:\ ; \
+    Remove-Item -Path innoextract.zip
+
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; \
+    Invoke-WebRequest $('https://github.com/docker/toolbox/releases/download/v{0}/DockerToolbox-{0}.exe' -f $env:DOCKER_VERSION) -OutFile 'dockertoolbox.exe' -UseBasicParsing
```

Comment:
>'https://github.com/docker/toolbox/releases/download/v{0}/DockerToolbox-{0}.exe' [](start = 24, length = 80)

Do you need to use the docker.exe containing Teja's fix for named pipe?

Reply:
I'm not sure, I need to talk to her more and look at the fix again, I wasn't being impacted by it.